src/main/scripts/src/app/shared/perspectives/documents/post-it/document/post-it-document.component.html

```diff
@@ -0,0 +1,46 @@
+<div class="post-it">
+  <!-- icon and name -->
+  <div class="header">
+    <span class="icon" title="{{ collection.name }}">
+      <i class="fa fa-fw {{ collection.icon }}" [style.color]="collection.color"></i>
+    </span>
+    <label class="name">{{ collection.name }}</label>
```

Comment:
Do we really want to show the collection name on each document post-it. Have not we agreed on showing it only when go over the icon with the mouse? I think it might be quite confusing for new users to see it everywhere. Maybe I would keep it only on the post-it which is used for adding new documents.